ex03.exs

```diff
@@ -61,8 +61,121 @@ defmodule Ex03 do
 
   """
 
```

Comment:
I think I've already commented on the complexity of this: Enum.chunk_every will do all this for you...